solidity/contracts/deposit/DepositLiquidation.sol

```diff
@@ -195,16 +195,15 @@ library DepositLiquidation {
         if (initiator == address(0)){
             initiator = address(0xdead);
         }
-        if (contractEthBalance > 1) {
+        if (contractEthBalance > valueToDistribute) {
```

Comment:
+ 1?

Reply:
derp, brain glitched on this, thought about it too... nice catch